compute/gpu/reference-content/kubernetes-gpu-time-slicing.mdx

```diff
@@ -0,0 +1,56 @@
+---
+meta:
+  title: NVIDIA GPU time-slicing with Kubernetes
+  description: This section provides information about NVIDIA GPU time-slicing with Kubernetes
+content:
+  h1: NVIDIA GPU time-slicing with Kubernetes
+  paragraph: This section provides information about NVIDIA GPU time-slicing with Kubernetes
+tags: gpu nvidia
+dates:
+  validation: 2023-04-25
+  posted: 2022-03-25
+categories:
+  - compute
+---
+
+NVIDIA GPUs are powerful hardware commonly used for Model training, deep learning, scientific simulations, and data processing tasks. On the other hand, Kubernetes (K8s) is a container orchestration platform that helps to manage and deploy containerized applications.
+
+Time slicing in the context of NVIDIA GPUs and Kubernetes refers to sharing a physical GPU among multiple containers or pods in a Kubernetes cluster.
+
+The technology involves partitioning the GPU's processing time into smaller intervals and allocating those intervals to different containers or pods. This technique allows multiple workloads to run on the same physical GPU, effectively sharing its resources while providing isolation between the different workloads.
+
+## Operational procedures of GPU time-slicing in Kubernetes
+
+Time-slicing NVIDIA GPUs with Kubernetes involves:
+* Dynamically allocating and sharing GPU resources among multiple containers or pods in a cluster.
+* Allowing each pod or container to use the GPU for a specific time interval before switching to another.
+* Efficiently using the available GPU capacity.
+
+This allows multiple workloads to utilize the GPU by taking turns in rapid succession. 
+
+* **GPU sharing:** Time-slicing involves sharing a single GPU among containers or pods by allocating small time intervals. Sharing is achieved by rapidly switching between different containers or pods, allowing them to use the GPU for a short duration before moving on to the next workload.
+* **GPU context switching:** GPU context switching refers to saving one workload's state, loading another's, and resuming processing. Modern GPUs are designed to handle context switching efficiently.
```

Comment:
I don't know if it is necessary to repeat "GPU context switching" after the colon here

Reply:
```suggestion
* **GPU context switching:** Refers to saving one workload's state, loading another's, and resuming processing. Modern GPUs are designed to handle context switching efficiently.
```